test/SILGen/initializers.swift

```diff
@@ -800,7 +800,7 @@ class ThrowDerivedClass : ThrowBaseClass {
   // CHECK:   try_apply [[UNWRAP_FN]]({{%.*}}) : $@convention(thin) (Int) -> (Int, @error any Error), normal [[NORMAL_BB:bb[0-9]+]], error [[ERROR_BB:bb[0-9]+]]
   //
   // Now we emit the call to the initializer. Notice how we return self back to
-  // its memory locatio nbefore any other work is done.
+  // its memory locations nbefore any other work is done.
```

Comment:
It really looks like this is meant to be

```suggestion
  // its memory location before any other work is done.
```

no?

Reply:
@adrian-prantl thx, right you are. Fixed!